consent/migrations/sql/cockroach/13.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+-- Fix performance issue of Admin API - Revoke Login Sessions
+CREATE INDEX hydra_oauth2_authentication_session_sub_idx ON hydra_oauth2_authentication_session (subject);
+
```

Comment:
This is missing keys used in e.g. MySQL / PostgreSQL. Is this by choice or accident?

Reply:
Sorry, I don't understand your question.
Because cockroach/12.sql didn't make subject as an index, I add it this time.
This should be useful for revoking login sessions respect to user's "subject".
Did I use the wrong cockroach syntax?

Reply:
Sorry, I figured it out. I should use `CREATE INDEX ON hydra_oauth2_authentication_session (subject);`, right?

Reply:
Sorry, what I meant was that the other migration files have a lot more indices added, such as:

```
CREATE INDEX hydra_oauth2_authentication_session_sub_idx ON hydra_oauth2_authentication_session (subject);
 CREATE INDEX hydra_oauth2_authentication_request_login_session_id_idx ON hydra_oauth2_authentication_request (login_session_id);
 CREATE INDEX hydra_oauth2_consent_request_login_session_id_idx ON hydra_oauth2_consent_request (login_session_id);
 CREATE INDEX hydra_oauth2_consent_request_login_challenge_idx ON hydra_oauth2_consent_request (login_challenge);
```

However, this file only has one key added (subject). I wanted to know if this is intentional (and for what reason) or not :)

Reply:
It is intentional, because cockroach db already have other indices added before :)

hydra_oauth2_authentication_request_login_session_id_idx -->  [consent/migrations/sql/cockroach/12.sql +42](https://github.com/ory/hydra/blob/9dbf8b538b3374a0293884b9d6a0663ab378648c/consent/migrations/sql/cockroach/12.sql#L42)
hydra_oauth2_consent_request_login_session_id_idx --> [consent/migrations/sql/cockroach/12.sql +22](https://github.com/ory/hydra/blob/9dbf8b538b3374a0293884b9d6a0663ab378648c/consent/migrations/sql/cockroach/12.sql#L22)
hydra_oauth2_consent_request_login_challenge_idx -->  [consent/migrations/sql/cockroach/12.sql +23](https://github.com/ory/hydra/blob/9dbf8b538b3374a0293884b9d6a0663ab378648c/consent/migrations/sql/cockroach/12.sql#L23)

Reply:
Ah right! Looks like the contributor already knew about this limitation :) Then this is good to go IMO!